src/components/Navigation/components/Section/tests/Section.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import {act} from 'react-dom/test-utils';
```

Comment:
What do you think to reexporting act in test-legacy instead of adding imports from `react-dom/test-utils`?

So we'd end up with

```
import {
  findByTestID,
  trigger,
  act,
  mountWithAppProvider,
} from 'test-utilities/legacy';

Reply:
I'm ok with that, it will clean up our imports. I'll make the change when I add a test later to bump `codecov`.